pallets/bridge_transfer/src/lib.rs

```diff
@@ -34,8 +33,8 @@ pub enum LotteryEvent {
 	OpenBox(u32, u32, Vec<u8>),
 }
 
-pub trait Config: system::Config + bridge::Config + pallet_mq::Config {
```

Comment:
Why the pallet_mq::Config be removed?

Reply:
It was removed accidental, already fixed.